syncthing/Info.plist

```diff
@@ -19,7 +19,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.12.1-1</string>
+	<string>1.13.0-1</string>
 	<key>CFBundleVersion</key>
 	<string>101200101</string>
```

Comment:
You need to set the CFBundleVersion to `101300001`

Reply:
thank you, update pushed